src/config.ts

```diff
@@ -741,6 +780,27 @@ export const FF_MIT_VOUCHER_ENABLED =
 export const FF_USER_AGE_LIMIT_ENABLED =
   process.env.FF_USER_AGE_LIMIT_ENABLED === "1";
 
+// IOLOGIN FF variable
+export const FF_IOLOGIN = pipe(
+  process.env.FF_IOLOGIN,
+  FeatureFlag.decode,
+  E.getOrElseW(() => FeatureFlagEnum.NONE)
+);
+
+export const IOLOGIN_USERS_LIST = pipe(
+  process.env.IOLOGIN_TEST_USERS,
+  CommaSeparatedListOf(FiscalCode).decode,
+  E.getOrElseW((err) => {
+    throw new Error(`Invalid IOLOGIN_TEST_USERS value: ${readableReport(err)}`);
+  })
+);
+
+export const IOLOGIN_CANARY_USERS_SHA_REGEX = pipe(
+  process.env.IOLOGIN_CANARY_USERS_REGEX,
+  NonEmptyString.decode,
+  E.getOrElse(() => "XYZ" as NonEmptyString)
```

Comment:
Is this an error?

Reply:
i followed the same default of another FF regex pattern declared later in the file
```typescript
export const FF_ROUTING_PUSH_NOTIF_CANARY_SHA_USERS_REGEX = pipe(
  process.env.FF_ROUTING_PUSH_NOTIF_CANARY_SHA_USERS_REGEX,
  NonEmptyString.decode,
  E.getOrElse((_) => "XYZ" as NonEmptyString)
);
```

should we change the default?

Reply:
It was used as a placehoder to allow undefined values. We can change it to another default, or even raise an error if not present.

Reply:
mmm, IMHO the `NonEmptyString` decode with a `getOrElse` is a useless overkill.

Reply:
It was necessary in case `FF_ROUTING_PUSH_NOTIF_CANARY_SHA_USERS_REGEX` could be `undefined`.
Since we are going to remove it asap, I think it's fine.